src/dbus/dbus_server_test.cpp

```diff
@@ -29,19 +29,42 @@
 #include "dbus/dbus_message_helper.hpp"
 #include "dbus/dbus_object.hpp"
 
-using namespace otbr::dbus;
-using namespace std::placeholders;
+using otbr::dbus::DBusMessageEncodeToVariant;
+using otbr::dbus::DBusMessageExtractFromVariant;
+using otbr::dbus::DBusMessageToTuple;
+using otbr::dbus::DBusObject;
+using otbr::dbus::DBusRequest;
+using std::placeholders::_1;
 
 class TestObject : public DBusObject
 {
 public:
     TestObject(DBusConnection *aConnection)
         : DBusObject(aConnection, "/org/otbr/testobj")
+        , mCount(0)
     {
         RegisterMethod("org.otbr", "Ping", std::bind(&TestObject::PingHandler, this, _1));
+        RegisterGetPropertyHandler("org.otbr", "Count", std::bind(&TestObject::CountGetHandler, this, _1));
```

Comment:
Do we prefer lambda to `std::bind`?

Reply:
We don't really have a preference now so maybe we can keep it as it is.